### PR TITLE
 sql-statements: add explanation for `explain format = "brief"`

### PR DESCRIPTION
**Pull request by time-and-fate**
### What is changed, added or deleted? (Required)

Add explanation for `explain format = "brief"`.
This functionality was introduced in v5.0 but remains undocumented.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/22677

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-explain.md [hunk 1]
```diff
@@ -183,7 +183,27 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
 4 rows in set (0.01 sec)
 ```
 
-如果未指定 `FORMAT`，或未指定 `FORMAT ="row"`，那么 `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [Understand the Query Execution Plan](https://pingcap.com/docs/dev/reference/performance/understanding-the-query-execution-plan/)。
+如果未指定 `FORMAT`，或指定 `FORMAT ="row"`，那么 `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [Understand the Query Execution Plan](https://pingcap.com/docs/dev/reference/performance/understanding-the-query-execution-plan/)。
```
Comment (TomShawn) [inline sql-statements/sql-statement-explain.md:186]:
```suggestion
如果未指定 `FORMAT`，或指定 `FORMAT ="row"`，那么 `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [TiDB 执行计划概览](/explain-overview.md)。
```

**Review by chrysan — APPROVED** [review]

File: sql-statements/sql-statement-explain.md [hunk 2]
```diff
@@ -183,7 +183,27 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
 4 rows in set (0.01 sec)
 ```
 
-如果未指定 `FORMAT`，或未指定 `FORMAT ="row"`，那么 `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [Understand the Query Execution Plan](https://pingcap.com/docs/dev/reference/performance/understanding-the-query-execution-plan/)。
```
Comment (TomShawn) [inline sql-statements/sql-statement-explain.md:186]: 能否先解释下什么是 FORMAT，字段解释是什么？

File: sql-statements/sql-statement-explain.md [hunk 3]
```diff
@@ -183,7 +183,27 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
...
+如果未指定 `FORMAT`，或指定 `FORMAT ="row"`，那么 `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [Understand the Query Execution Plan](https://pingcap.com/docs/dev/reference/performance/understanding-the-query-execution-plan/)。
+
+如果指定 `FORMAT ="brief"`，那么 `EXPLAIN` 语句将会在未指定 `FORMAT` 时的输出结果的基础上，简化其中的算子 ID：
```
Comment (TomShawn) [inline sql-statements/sql-statement-explain.md:188]:
```suggestion
如果在 `EXPLAIN` 中指定了 `FORMAT ="brief"`，那么 `EXPLAIN` 语句输出结果中的算子 ID，较之未指定 `FORMAT` 时输出结果的算子 ID 更为简化：
```

File: sql-statements/sql-statement-explain.md [hunk 4]
```diff
@@ -183,7 +183,27 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
...
+
+{{< copyable "sql" >}}
+
+```sql
+EXPLAIN FORMAT = "brief" DELETE FROM t1 WHERE c1=3;
```
Comment (TomShawn) [inline sql-statements/sql-statement-explain.md:193]: 是否可以给一个不指定 FORMAT = "brief" 时的输出结果作为对比呢？

Comment (time-and-fate) [reply sql-statements/sql-statement-explain.md:193]: 这个是和上面使用的同样的 SQL 作为例子，可以对比上面的结果。
https://github.com/pingcap/docs-cn/blob/4d705a543d212ba59de0f9fa4f8d2639b1b389a8/sql-statements/sql-statement-explain.md?plain=1#L170-L184

File: sql-statements/sql-statement-explain.md (hunk 2 above)
Comment (time-and-fate) [reply sql-statements/sql-statement-explain.md:186]: 已更新~

File: sql-statements/sql-statement-explain.md [hunk 5]
```diff
@@ -183,9 +183,31 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
 4 rows in set (0.01 sec)
 ```
 
-如果未指定 `FORMAT`，或未指定 `FORMAT ="row"`，那么 `EXPLAIN` 语句将以表格格式输出结果。更多信息，可参阅 [Understand the Query Execution Plan](https://pingcap.com/docs/dev/reference/performance/understanding-the-query-execution-plan/)。
+使用 `FORMAT = "xxx"` 语法可以指定输出的内容和格式。
```
Comment (TomShawn) [inline sql-statements/sql-statement-explain.md:186]:
```suggestion
在 `EXPLAIN` 中使用 `FORMAT = "xxx"` 语法可以指定输出的内容和格式。
```

File: sql-statements/sql-statement-explain.md [hunk 6]
```diff
@@ -183,9 +183,31 @@ EXPLAIN DELETE FROM t1 WHERE c1=3;
...
++-------------------------+---------+-----------+---------------+--------------------------------+
+| Delete                  | N/A     | root      |               | N/A                            |
+| └─TableReader           | 0.00    | root      |               | data:Selection                 |
+|   └─Selection           | 0.00    | cop[tikv] |               | eq(test.t1.c1, 3)              |
+|     └─TableFullScan     | 3.00    | cop[tikv] | table:t1      | keep order:false, stats:pseudo |
++-------------------------+---------+-----------+---------------+--------------------------------+
+4 rows in set (0.001 sec)
+```
+
+除 MySQL 标准结果格式外，TiDB 还支持 DotGraph。需按照下列所示指定 `FORMAT = "dot"`：
```
Comment (TomShawn) [inline sql-statements/sql-statement-explain.md:210]:
```suggestion
除 MySQL 标准结果格式外，TiDB 还支持 DotGraph，需要在 `EXPLAIN` 中指定 `FORMAT = "dot"`，示例如下：
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**